src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -1055,10 +1052,19 @@ private bool isIntrinsicType(CORINFO_CLASS_STRUCT_* classHnd)
             return type.IsIntrinsic;
         }
 
+        private MethodSignatureFlags PlatformDefaultUnmanagedCallingConvention()
+        {
+            return _compilation.TypeSystemContext.Target.IsWindows ?
+                MethodSignatureFlags.UnmanagedCallingConventionCdecl : MethodSignatureFlags.UnmanagedCallingConventionCdecl;
```

Comment:
@jkotas is it intended, that both conditional expressions are `Cdecl`? `IsWindows` is a redundant check then. Perhaps one should be `stdcall`.

Reply:
Good catch! Thank you!

#376